src/commands/help.js

```diff
@@ -67,9 +73,26 @@ class TopicHelpPresenter extends TopicBase {
     if (topic) this.constructor.topic = topic
   }
 
-  async help (args: string[], matchedCommand?: ?Class<Command<*>>) {
+  async help (args: string[], matchedCommand?: ?Class<Command<*>>, matchedNamespace?: any) {
```

Comment:
it would be nice if matchedNamespace wasn't any type here, would help me understand what it is for sure. Is it a plugin?